pkg/doc/verifiable/credential.go

```diff
@@ -319,7 +365,7 @@ func validate(data []byte, schema *CredentialSchema, opts *credentialOpts) error
 	loader := gojsonschema.NewStringLoader(string(data))
 	result, err := gojsonschema.Validate(schemaLoader, loader)
 	if err != nil {
-		return fmt.Errorf("Validation of Verifiable Credential failed: %w", err)
```

Comment:
Why %v instead of %w? Is wrapping the error not appropriate here?
(and the other similar changes in this PR).

Reply:
agree - I will revert it!

Reply:
Fixed